Makefile

```diff
@@ -59,6 +59,7 @@ clean-docker-containers: ## Clean up any remaining docker containers
 format:
 	ruff check --select I --fix .
 	ruff format .
+	ruff check
```

Comment:
Just tweaking `ruff` a bit here. CI was outputting some errors from linting categories we weren't checking for previously.